config/game_options.txt

```diff
@@ -163,14 +163,14 @@ ALLOW_AI_MULTICAM
 ## AWAY MISSIONS ###
 
 ## Uncomment to load one of the missions from awaymissionconfig.txt or away_missions/ at roundstart.
-#ROUNDSTART_AWAY
+ROUNDSTART_AWAY
 
 ## How long the delay is before the Away Mission gate opens. Default is half an hour.
 ## 600 is one minute.
-GATEWAY_DELAY 18000
+GATEWAY_DELAY 60
 
 ## The probability of the gateway mission being a config one
-CONFIG_GATEWAY_CHANCE 0
+CONFIG_GATEWAY_CHANCE 100
```

Comment:
This wont matter as far as test-merging since the config doesn't get overwritten by repository changes but remove this before you un-DNM this.